app/src/androidTest/java/com/github/sdpcoachme/messaging/ChatActivityTest.kt

```diff
@@ -85,8 +91,7 @@ class ChatActivityTest {
 
     @Test
     fun startingElementsArePresent() {
-        val chatIntent = Intent(ApplicationProvider.getApplicationContext(), ChatActivity::class.java)
-        chatIntent.putExtra("toUserEmail", toUser.email)
+        val chatIntent = defaultIntent.putExtra("toUserEmail", toUser.email)
```

Comment:
This line is duplicated in all tests except errorHandlerIsLaunchedIfToUserEmailNotPassedInIntentExtra so I would recommend to put the extra (email) directly when declaring the defaultIntent at line 51 and create a special intent without email for the specific test just mentioned.

Reply:
I've modified this, thanks for the comment.